src/components/Wizard/CreateVmWizard/providers/index.js

```diff
@@ -21,6 +27,14 @@ const getProviderHelp = basicSettings => {
 };
 
 export const importProviders = (basicSettings, operatingSystems, WithResources, k8sCreate, k8sGet, k8sPatch) => ({
+  [PROVIDER_VMWARE_NAMESPACE_ALERT_KEY]: {
+    id: 'vcenter-namespace-alert',
+    type: CUSTOM,
+    title: ' ',
+    CustomComponent: () => <Alert type="warning">Please select a namespace.</Alert>,
+    isVisible: basicVmSettings =>
+      isImageSourceType(basicVmSettings, PROVISION_SOURCE_IMPORT) && !settingsValue(basicSettings, NAMESPACE_KEY),
```

Comment:
There was plan to make this requirement more generic, ie. by disabling subsequent dropdowns if `namespace` is not selected. @rawagner, any news?

Reply:
no news, havent had time to look at it. Here is a design https://docs.google.com/document/d/1ks4H-ImD-r0s5eV4FsP_73kRdAP6sKJTRbj8nClpyp0/edit#heading=h.o4npgyz4fz5s which shows that by disabling dropdowns/other fields we would 'guide' user to correct flow.